<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="adminpic">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black;
height: 25px;
  width: 25px;
  border-radius: 50%;
  display: inline-block;</string>
    </property>
    <property name="text">
     <string>admin</string>
    </property>
   </widget>
   <widget class="QLabel" name="instructorpic">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black;
height: 25px;
  width: 25px;
  border-radius: 50%;
  display: inline-block;</string>
    </property>
    <property name="text">
     <string>instructor</string>
    </property>
   </widget>
   <widget class="QLabel" name="studentpic">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black;
height: 30px;
  width: 30px;
  border-radius: 50%;
  display: inline-block;</string>
    </property>
    <property name="text">
     <string>student</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>381</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
</string>
    </property>
    <property name="title">
     <string>SIGN IN</string>
    </property>
    <widget class="QPushButton" name="sign_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Sign</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>show</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>261</width>
       <height>102</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="ID">
          <property name="text">
           <string> ID              </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="idprint">
          <property name="styleSheet">
           <string notr="true">width: 100%;
  box-sizing: border-box;
  border: 2px solid #ccc;
  border-radius: 1px;
  color:black;
  font-size: 16px;
  background-color: white;
  background-position: 10px 10px; 
  background-repeat: no-repeat;
  padding: 10px 15px 10px 35px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="password">
          <property name="text">
           <string>PASSWORD</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordprint">
          <property name="styleSheet">
           <string notr="true">width: 50%;
  box-sizing: border-box;
  border: 2px solid #ccc;
  border-radius: 1px;
  font-size: 16px;
  background-color: white;
  background-position: 10px 10px; 
  background-repeat: no-repeat;
  padding: 10px 15px 10px 35px;
color:black;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="id_check">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="instructor_button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
  border: none;
  color: red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="student_button">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
  border: none;
  color: red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
  border: none;
  color: red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="background__of__all">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>381</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black;
height: 20px;
  width: 20px;
  border-radius:40%;
  display: inline-block;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
font: 11pt &quot;MS Shell Dlg 2&quot;;
font-weight:bold;</string>
    </property>
    <property name="text">
     <string>SIGN IN AS</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_instructor">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="text_student">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="text_admin">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="background_of_all2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>381</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="back1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black;
height: 35px;
  width: 35px;
  border-radius: 60%;
  display: inline-block;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="back2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black;
height: 35px;
  width: 35px;
  border-radius: 60%;
  display: inline-block;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="back3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black;
height: 35px;
  width: 35px;
  border-radius: 60%;
  display: inline-block;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="back_all">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>2</y>
      <width>651</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>back_all</zorder>
   <zorder>background_of_all2</zorder>
   <zorder>background__of__all</zorder>
   <zorder>back3</zorder>
   <zorder>back2</zorder>
   <zorder>back1</zorder>
   <zorder>adminpic</zorder>
   <zorder>studentpic</zorder>
   <zorder>groupBox</zorder>
   <zorder>instructorpic</zorder>
   <zorder>instructor_button</zorder>
   <zorder>student_button</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>label</zorder>
   <zorder>text_instructor</zorder>
   <zorder>text_student</zorder>
   <zorder>text_admin</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>683</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
